pom.xml

```diff
@@ -44,10 +44,10 @@ THE SOFTWARE.
   </licenses>
     
   <properties>
-    <jenkins.version>1.609.3</jenkins.version>
+    <jenkins.version>1.642.3</jenkins.version>
```

Comment:
As mentioned in the commit message, we needed to bump dependency versions to get the latest `durable-task-step` for `PlaceholderTask.getNode()`.